apps/heft/src/plugins/JestPlugin/JestPlugin.ts

```diff
@@ -145,6 +150,56 @@ export class JestPlugin implements IHeftPlugin {
     clean.properties.pathsToDelete.add(cacheFolder);
   }
 
+  private async _getJestReporters(
```

Comment:
Would it make sense to add a simple test project (`build-tests/heft-jest-reporters-test/`?) that illustrates a `jest.config.json` that is actually using this feature?  I'm not really interested assertions to validate that it works properly, but rather a concrete example that the documentation can refer to.  And something Heft developers can try building with if they need to make changes to this code.